datasets/cifar100/README.md

```diff
@@ -0,0 +1,149 @@
+```
+### Data Fields
+ 
+- `img`:  32x32x3 image
+- `fine_label`: a classification label, with possible values including `apple`, `aquarium_fish`, `baby`, `bear`, `beaver` and more.
+- `coarse_label`: a classification label, with possible values including `aquatic_mammals`, `fish`, `flowers`, `food_containers`, `fruit_and_vegetables` and more.
```

Comment:
It would be nice to have a way to show the mapping between the label ids (0, 1, 2, ...) and names (apple, aquarium_fish, ...) since this is actually the label id that is returned with `dataset[0]["fine_grained"]`.

Can you add the full list with both ids and names here ? Or maybe redirect to a page that has the full list ?

Reply:
I can make a list and then store the label names along with the ids. Does that sound good?